src/Akka.Persistence.MongoDb.Hosting/MongoDbSnapshotOptions.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Text;
+using Akka.Configuration;
+using Akka.Hosting;
+using Akka.Persistence.Hosting;
+
+#nullable enable
+namespace Akka.Persistence.MongoDb.Hosting;
+
+public class MongoDbSnapshotOptions : SnapshotOptions
+{
+    private static readonly Config Default = MongoDbPersistence.DefaultConfiguration()
+        .GetConfig(MongoDbSnapshotSettings.SnapshotStoreConfigPath);
+
+    public MongoDbSnapshotOptions() : this(true)
+    {
+    }
+
+    public MongoDbSnapshotOptions(bool isDefault, string identifier = "mongodb") : base(isDefault)
+    {
+        Identifier = identifier;
+        AutoInitialize = true;
+    }
+
+    /// <summary>
+    /// Connection string used to access the MongoDb, also specifies the database.
+    /// </summary>
+    public string ConnectionString { get; set; } = "";
+
+    /// <summary>
+    /// Name of the collection for the event journal or snapshots
+    /// </summary>
+    public string Collection { get; set; } = "SnapshotStore";
+
+    /// <summary>
+    /// Transaction
+    /// </summary>
+    public bool UseWriteTransaction { get; set; }
```

Comment:
Same as above

Reply:
Explicit default has been set - Confirmed it matches reference conf from `snapshot-store ` section.

https://github.com/akkadotnet/Akka.Persistence.MongoDB/blob/1f98a99f2865995a1497148592d2dd9e11721535/src/Akka.Persistence.MongoDb/reference.conf#L49C4-L49C4